.travis.yml

```diff
@@ -37,8 +37,10 @@ jobs:
           branch: master
           condition:  $TRAVIS_OS_NAME == linux
         provider: script
-        skip_cleanup: true
         script: npm run semantic-release
 
+after_script:
+  - cat test-worker/build/test-results/test-worker/test.log
```

Comment:
This is useful for seeing test-worker error details.

Reply:
this could clog up the travis report. I would recommend only displaying the logs for the failed tests

Reply:
It's not a problem, see for example https://travis-ci.com/github/adobe/asset-compute-sdk/jobs/392950549 (scroll to the very bottom).

Travis is smart and collapses it by default. I had a couple of builds now (x the nodejs variants) and there was no indication of an issue.

Reply:
sounds good.

however, it is still pretty hard to scroll through those logs (since we have so many test cases)...this is probably more general for our test worker logs but it would be nice to have separate logs for only the failing test cases

Reply:
yeah, but you can search for the test case name which works pretty well IMO.

When running/debugging locally run `... test-worker <test-case-folder>` and it will just run that test. then a `cat build/test.log` will be limited to just that test.